scripts/cluster_network.py

```diff
@@ -383,12 +384,14 @@ def locate_bus(coords, co):
         point = Point(coords["x"], coords["y"])
 
         try:
-            return gdf_co[gdf_co.contains(point)]["GADM_ID"].item()
+            gadm_id = gdf_co[gdf_co.contains(point)]["GADM_ID"].item()
+            return three_2_two_digits_country(gadm_id[:3]) + gadm_id[3:]
 
         except ValueError:
-            return gdf_co[
+            gadm_id = gdf_co[
                 gdf_co.geometry == min(gdf_co.geometry, key=(point.distance))
             ]["GADM_ID"].item()
+            return three_2_two_digits_country(gadm_id[:3]) + gadm_id[3:]
```

Comment:
I think it it's fine for now.  Just flagging its likely that we move from two letter to three letter code in future.
Just to clarify, whats the output of: 
- `gadm_id[3:]`
- `gadm_id[:3]`
- three_2_two_digits_country(gadm_id[:3])

Reply:
The GADM-id's are strings, which typically consist out of the three-digit country abbreviation, a dot and the numerical part which specifies the region according to the GADM level 1. One example would be the id of the Brazilian federal state Acre, which is "BRA.1_1".
- gadm_id[3:] contains the dot and the numeric part of the gadm id 
- gadm_id[:3] contains the three-digit id of the country
- three_2_two_digits_country(gadm_id[:3]) contains the two-digit equivalent of the country id e.g. BR for Brazil

Reply:
I think `gadm_id[3:]` would break for other cases. The id can be larger than 3 numbers. See:
![image](https://user-images.githubusercontent.com/61968949/230158778-0799196e-5e85-4f84-ac13-a291f3d48200.png)
Please, revise this part and use a Python string split instead: https://www.w3schools.com/python/ref_string_split.asp

Reply:
Slicing the id-string using `gadm_id[3:]` returns all characters from the third one until the last one. So theoretically, even for larger GADM id's as the ones with second level resolution like "NGA.18.23_1", it should work, as long as the alphabetic country code only consists of three letters, which should be always the case for GID's following the ISO norm.

For more information, please refer to the bottom paragraph paragraph at:
https://gadm.org/metadata.html

Reply:
@cschau-ieg you are right sorry. I think it's fine. Let's merge this.